api.md

```diff
@@ -710,6 +731,76 @@ SELECT set_number_partitions('conditions', 2, 'device_id');
 
 ---
 
+## show_chunks() [](show_chunks)
+Get list of chunks associated with hypertables.
+
+#### Optional Arguments [](show_chunks-optional-arguments)
+
+Function accepts the following arguments. These arguments have
+the same semantics as the `drop_chunks` [function][drop_chunks].
+
+|Name|Description|
+|---|---|
+| `older_than` | Specification of cut-off point where any full chunks older than this timestamp should be shown. |
+| `newer_than` | Specification of cut-off point where any full chunks newer than this timestamp should be shown. |
+
+The `older_than` and `newer_than` parameters can be specified in two ways:
+
+- **interval type**: The cut-off point is computed as `now() -
+    older_than` and similarly `now() - newer_than`.  An error will be returned if an INTERVAL is supplied
+    and the time column is not one of a TIMESTAMP, TIMESTAMPTZ, or
+    DATE.
+
+- **timestamp, date, or integer type**: The cut-off point is
+    explicitly given as a TIMESTAMP / TIMESTAMPTZ / DATE or as a
+    SMALLINT / INT / BIGINT. The choice of timestamp or integer should
+    generally follow the type of the hypertable's time column.
+
+When both arguments are used, the function returns the intersection of the resulting two ranges. For example,
+specifying `newer_than => 4 months` and `older_than => 3 months` will show all full chunks that are between 3 and
+4 months old. Similarly, specifying `newer_than => '2017-01-01'` and `older_than => '2017-02-01'` will show
+all full chunks between '2017-01-01' and '2017-02-01'. Specifying parameters that do not result in an overlapping
+intersection between two ranges will result in an error.
+
+#### Sample Usage [](show_chunks-examples)
+
+Get list of all chunks. Returns 0 if there are no hypertables:
+```sql
+SELECT show_chunks();
```

Comment:
Given at least one example output so the reader knows the expected format of it?

Reply:
Added